ITC205_BaseLine_Code/src/ReturnBookControl.java

```diff
@@ -59,7 +59,7 @@ public void scanningComplete() {
 		if (!state.equals(CONTROL_STATE.READY)) {
 			throw new RuntimeException("ReturnBookControl: cannot call scanningComplete except in READY state");
 		}	
-		ui.setState(ReturnBookUI.UI_STATE.COMPLETED);		
+		ui.SetState(ReturnBookUI.UiState.COMPLETED);		
 	}
 
 

```

Comment:
D2 - delete blank line - line 65

Reply:
Will amend ReturnBookControl in alternate branch. This is the next branch to be amended.